src/slurmrestd_ops.py

```diff
@@ -0,0 +1,333 @@
+# Copyright 2024 Omnivector, LLC.
+# See LICENSE file for licensing details.
+"""This module provides the SlurmManager."""
+
+import logging
+import os
+import shlex
+import subprocess
+from base64 import b64decode
+from pathlib import Path
+
+import charms.operator_libs_linux.v0.apt as apt  # type: ignore [import-untyped]
+import charms.operator_libs_linux.v1.systemd as systemd  # type: ignore [import-untyped]
+import distro
+
+logger = logging.getLogger()
+
+
```

Comment:
You can have logger and the local constant together here

```suggestion
```

Reply:
Just going to leave as black likes it for now. We can change it later to your liking ... I don't mind.